### ISSUE TITLE
Nanopool setup

### ISSUE THREAD

**Issue by Faxoxpl** [UnamSanctam/SilentCryptoMiner#2209]
{
    "url": "stratum://`0xc40B69602C869185A78a7b1f56bFd1A532358F27`.{COMPUTERNAME}@etc-eu1.nanopool.org:10100",
    "algo": "etchash",
    "max-gpu": 30,
    "idle-wait": 5,
    "idle-gpu": 100,
    "stealth-targets": "",
    "kill-targets": "",
    "stealth-fullscreen": false
}
it will be work?

**Comment by UnamSanctam** [comment]
Yes that's correct, though there are a lot more profitable things to mine than ETC: https://whattomine.com/
